itests/run.sh

```diff
@@ -7,30 +7,57 @@ if [ ! -d "target/dependency" ]; then
     mvn package dependency:copy-dependencies
 fi
 
+BACKEND=$1
+
+if [ "$BACKEND" == "ispn" ]
+then
+    JAVA_OPTS="$JAVA_OPTS -Dhawkular-alerts.backend=ispn -Dhawkular.data=$(pwd)/target/ispn-itests"
+fi
+
 JGROUPS_BIND_ADDR="127.0.0.1"
 JAVA_OPTS="$JAVA_OPTS -Xmx64m -Xms64m"
 JAVA_OPTS="$JAVA_OPTS -Djava.net.preferIPv4Stack=true -Djgroups.bind_addr=${JGROUPS_BIND_ADDR}"
 JAVA_OPTS="$JAVA_OPTS -Dmail.smtp.port=2525"
 JAVA_OPTS="$JAVA_OPTS -Dlog4j.configurationFile=src/test/resources/log4j2.xml"
 # JAVA_OPTS="$JAVA_OPTS -agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=y"
 
-CLUSTER=$1
+# [lponce] Ispn needs to activate a service from lucene jar and for some reason in standalone mode jar needs to be placed first in classpath
```

Comment:
Yeah, it is, I spent some time with ispn folks debugging this. It is a thing in how the java service providers work with urls, but this workaround works well.
Note, this is not the final distribution for .next, I will submit a PR with a dist/ folder which will simplify some of this configurations. This one is just for itests.